java/src/jmri/jmrit/throttle/ThrottlesPreferencesControlsSettingsPane.java

```diff
@@ -455,7 +455,7 @@ public void setDirty(boolean b) {
             setEditable(false);
             String text="";
             if (v[0]!=0) {
-                text +=  KeyEvent.getKeyModifiersText(v[0])+" + ";
+                text +=  (KeyEvent.getKeyModifiersText(v[0]).isEmpty()?KeyEvent.getModifiersExText(v[0]):KeyEvent.getKeyModifiersText(v[0]).isEmpty()) +" + ";
```

Comment:
No need to change, but my personal view is that this line would be easier to read with:
```
                text += (KeyEvent.getKeyModifiersText(v[0]).isEmpty()
                        ? KeyEvent.getModifiersExText(v[0])
                        : KeyEvent.getKeyModifiersText(v[0]).isEmpty())
                        + " + ";
```

Reply:
What is the purpose of `: KeyEvent.getKeyModifiersText(v[0]).isEmpty())`, the second part of the `? :` operator? It seems to me that this will always be `false`. So that this text is really:
```
                text += (KeyEvent.getKeyModifiersText(v[0]).isEmpty()
                        ? KeyEvent.getModifiersExText(v[0])
                        : false)
                        + " + ";
```

Reply:
By looking at the code again, it seems that the code should be:
```
                text += (KeyEvent.getKeyModifiersText(v[0]).isEmpty()
                        ? KeyEvent.getModifiersExText(v[0])
                        : KeyEvent.getKeyModifiersText(v[0]))
                        + " + ";
```

Reply:
you're right! fixing now.